src/SUMMARY.md

```diff
@@ -297,19 +292,37 @@
   - [Dining Philosophers](exercises/concurrency/dining-philosophers-async.md)
   - [Broadcast Chat Application](exercises/concurrency/chat-app.md)
 
+# WebAssembly
+
+---
+
+- [WebAssembly basics](webassembly.md)
+- [Load a WASM module](webassembly/load-wasm-module.md)
```

Comment:
Can you please go through the whole PR and replace all `WASM` (in prose) by `Wasm`?